src/hu/elte/txtuml/stdlib/util/Scanner.java

```diff
@@ -0,0 +1,76 @@
+package hu.elte.txtuml.stdlib.util;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+
+import hu.elte.txtuml.api.model.external.ExternalClass;
+
+public interface Scanner extends ExternalClass{
+	/*
+	Constructs a new Scanner that produces values scanned from the specified file.
+	 */
+	public void scanner(File file);
```

Comment:
/*
It's done, i've changes the File into a String value which specifies the path of the file
*/
@Override
	public void Scanner(String source) {
			sc = new Scanner(source);
	}